### PR TITLE
Add multi-channel support to `updateBrowser.js`

### PR DESCRIPTION
**Pull request by bsclifton**
Add multi-channel support to `updateBrowser.js`. Pull common code into `common.js`

Fixes https://github.com/brave/browser-laptop/issues/11795

Auditors: @aekeus

### CODE REVIEW

File: bin/uploadBrowser.js [hunk 1]
```diff
@@ -72,7 +72,7 @@ var OS_IDENTIFIER = 2
 // Recipe tuples containing local relative paths to files, key locations on S3, and an os identifier
 var recipes = [
   // Linux
-  ['dist/Brave.tar.bz2', 'multi-channel/releases/CHANNEL/VERSION/linux64', 'linux'],
+  ['dist/Brave{{channelName}}.tar.bz2', 'multi-channel/releases/CHANNEL/VERSION/linux64', 'linux'],
```
Comment (bsclifton) [inline bin/uploadBrowser.js]: This is one I missed last time... not important though, since Linux users can't upgrade without apt/yum/dnf

**Review by darkdh — APPROVED** [review]
lgtm

**Comment by bsclifton** [comment]
@aekeus can you please review this one? 😄

**Comment by bsclifton** [comment]
Before anyone starts reviewing / rereviewing, please let me run some tests to make sure this works great as expected 😄 Maybe this can be reviewed again next week

**Comment by bsclifton** [comment]
I updated Jenkins to use this code and did a Beta build- update worked great! 😄 

Updated from: `0.20.14` to `0.20.19`

Here are logs from the update:
```
2018-01-23T05:08:14.056Z - checkForUpdates
2018-01-23T05:08:14.056Z - lastCheckYMD = 2018-01-22
2018-01-23T05:08:14.056Z - lastCheckWOY = 19
2018-01-23T05:08:14.056Z - lastCheckMonth = 1
2018-01-23T05:08:14.056Z - weekOfInstallation= 2017-12-18
2018-01-23T05:08:14.056Z - firstCheckMade = true
2018-01-23T05:08:14.056Z - promoCode = none
2018-01-23T05:08:14.056Z - https://laptop-updates.brave.com/1/releases/beta/0.20.14/osx?daily=false&weekly=false&monthly=false&first=false&woi=2017-12-18&ref=none&accept_preview=false
2018-01-23T05:08:14.455Z - Metadata: {"version":"0.20.19","name":"Brave 0.20.19","pub_date":"2018-01-23T05:10:43.282Z","notes":"More details: https://github.com/brave/browser-laptop/releases/tag/v0.20.19beta\n\nMore details: https://github.com/brave/browser-laptop/releases/tag/v0.20.17\nbeta\n\nMore details: https://github.com/brave/browser-laptop/releases/tag/v0.20.16\nbeta\n\nMore details: https://github.com/brave/browser-laptop/releases/tag/v0.20.15\nbeta","preview":false,"url":"https://brave-download.global.ssl.fastly.net/multi-channel/releases/beta/0.20.19/osx/Brave-Beta-0.20.19.zip"}
2018-01-23T05:08:14.672Z - update downloading
2018-01-23T05:08:37.070Z - update downloaded
2018-01-23T05:08:37.070Z - Release notes : More details: https://github.com/brave/browser-laptop/releases/tag/v0.20.19beta

More details: https://github.com/brave/browser-laptop/releases/tag/v0.20.17
beta

More details: https://github.com/brave/browser-laptop/releases/tag/v0.20.16
beta

More details: https://github.com/brave/browser-laptop/releases/tag/v0.20.15
beta
2018-01-23T05:08:37.070Z - Release date: Brave 0.20.19
2018-01-23T05:08:37.070Z - Update URL: Wed Dec 31 1969 17:00:00 GMT-0700 (MST)
```

**Pull request merged**